cmd/ttn-lw-cli/internal/api/grpc.go

```diff
@@ -107,6 +125,34 @@ func GetDialOptions() (opts []grpc.DialOption) {
 	if withDump {
 		opts = append(opts, grpc.WithChainUnaryInterceptor(requestInterceptor))
 	}
+
+	callOpts := []grpc_retry.CallOption{
+		grpc_retry.WithBackoffContext(
+			func(ctx context.Context, attempt uint) time.Duration {
+				logger := log.FromContext(ctx)
+				logger.Debugf("Failed request, retry attempt %v", attempt)
```

Comment:
This is before getting the feedback on the use of jitter but I'm rethinking on how to log this information.

My current idea is to do something like:
`				logger.WithField("attempt", attempt).Debug("Failed request, waiting approximately %v until next attempt", attempt)`

Would like to ask if it is appropriate to leave the `approximately` in the log to give an idea about the jitter.